core/src/toast/pane/pane.module.css

```diff
@@ -2,7 +2,9 @@
 	display: flex;
 	align-items: center;
 	position: relative;
+}
 
+.default {
 	width: max-content;
 	max-width: 288px;
```

Comment:
I don't think "width" style should be here. Even though it's technically a visual styling, it should not be a part of the extensible API. The developer of Moai's themes (like banhmi) should not need to change the width value